unittest/client/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+############################################################################
+#
+# Copyright (C) 2023 Advanced Driver Information Technology Joint Venture GmbH
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#               http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+############################################################################
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
+cmake_policy(SET CMP0046 OLD)
+
```

Comment:
The warning messages when configuring of cmake.
https://github.com/COVESA/wayland-ivi-extension/pull/165/commits/65fa39b1b1edf8806f9bbfacaea07fae12347b26